Payments/UserInterface/01a_ListAuthorizations.md

```diff
@@ -0,0 +1,142 @@
+    If you click the checkbox in the header, all entries in the list are selected. The editing toolbar shows all actions you can perform on several entries.
+- ![Search](../../Assets/Icons/Search.png "[Search]") (Search)   
+    Click this button to display the search bar and search for entries.
+
+- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)   
+    Click this button to update the entries.
+
+- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
+    Click this button to display the columns bar and to customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.   
+    > [Info] It is recommended to add the following columns to your view to get the required information and to better filter and sort the entries: Status ID, Status information (if applicable). 
+- ![Filter](../../Assets/Icons/Filter.png "[Filter]") Filter (x)   
+    Click this button to display the filter bar and customize the active filters. The *x* indicates the number of filters that are currently active.
+
+The list displays all authorization transactions. Depending on the payment service provider, the displayed columns vary. All fields are read-only. The following standard columns are available.
+<!----Hallo Stefan, hier sollten alle Standard-Felder mit Erklärungen stehen, ist die Liste vollständig?--->
+- *Charges intended use*   
+   Purpose of the fee   
+- *Original currency*   
+   Currency in which the product(s) were paid   
+- *Original currency amount*   
+   Invoice amount in the original currency   
+
+
+
+## Authorizations &ndash; Attributes
+
+*Payments > Payments and Captures > Tab LIST > Select an authorization ticket > Tab Attributes*   
+
+
+![LIST &ndash; Attributes](../../Assets/Screenshots/Payments/Authorizations/AuthorizationAttributes.png "[LIST &ndash; Attributes]")
+
+In the left margin column, all available attribute groups are displayed. Click a group to display the attributes that are assigned to this group on the right side of the *Attributes* tab. If the authorization transaction contains attributes that are not assigned, the *Unassigned group* attribute group is automatically displayed in the left margin column.
+
+- ![Folders](../../Assets/Icons/Folders01.png "[Folders]") (Folders)  
+    Attribute group that contains attribute sub-groups. Click the attribute group or the arrow left to the attribute group to expand the group and to display the attribute sub-groups. The following sub-groups are available:
+    - *[Payment service provider] Incoming Payment Base Data*   
+      This folder contains attributes that are specific for the payment service provider who has sent this authorization ticket.
+    - *Incoming Payment Base Data*   
+      This folder contains general fields that Actindo needs to query payments data. These data support standard functions of Actindo. 
+
```

Comment:
Sind diese Ordner immer da oder ist es variabel, abhängig von Treiber? Gibt es andere Möglichkeiten? Evtl. kann man etwas wie "The following sub-groups are available by default:" schreiben.

Reply:
das wäre doppelt gemoppel